scripts/exregional_nexus_emission.sh

```diff
@@ -327,14 +327,32 @@ else
   fi
 fi
 POST_STEP
+
+# 
+#-----------------------------------------------------------------------
+#
+# make nexus output pretty and move to INPUT_DATA directory
+#
+#-----------------------------------------------------------------------
+#
+python3 ${ARL_NEXUS_DIR}/utils/python/make_nexus_output_pretty.py --src ${DATA}/NEXUS_Expt_split.nc --grid ${DATA}/grid_spec.nc -o ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.NEXUS_Expt_split.${nspt}.nc -t ${DATA}/HEMCO_sa_Time.rc
+export err=$?
+if [ $err -ne 0 ]; then
+  message_txt="Call to python script \"make_nexus_output_pretty.py\" failed."
+  if [ "${RUN_ENVIR}" = "community" ]; then
+    print_err_msg_exit "${message_txt}"
+  else
+    err_exit "${message_txt}"
+  fi
+fi
 #
 #-----------------------------------------------------------------------
 #
 # Move NEXUS output to INPUT_DATA directory.
 #
 #-----------------------------------------------------------------------
 #
-mv ${DATA}/NEXUS_Expt_split.nc ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.NEXUS_Expt_split.${nspt}.nc
+#mv ${DATA}/NEXUS_Expt_split.nc ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.NEXUS_Expt_split.${nspt}.nc
```

Comment:
If this line is not necessary, please remove this line with the description (Lines 348-355).